src/Acuminator/Acuminator.Utilities/Acuminator.Utilities.csproj

```diff
@@ -53,15 +53,6 @@
     <Reference Include="Microsoft.CodeAnalysis.Workspaces, Version=1.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@saloponov please finish pull request for ATR-355 first and merge it into this this pull request.